src/cmds/restic/cmd_backup.go

```diff
@@ -28,6 +29,10 @@ The "backup" command creates a new snapshot and saves the files and directories
 given as the arguments.
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if backupOptions.Stdin && backupOptions.FilesFrom == "-" {
```

Comment:
Hi there!
Question: What about `backupOptions.FilesFrom == "/dev/stdin"`? Shouldn't it be checked some way?
Cheers!

Reply:
No, because that'll work out of the box. Restic opens `/dev/stdin` as a regular file and reads from it, nothing special here.

Reply:
So I assume it is still valid to call restic with`--stdin` and `--files-from /dev/stdin`. Is it right?

Reply:
I haven't tried it, but I think that will work just fine.